Sources/Tracing/TracerProtocol.swift

```diff
@@ -59,6 +59,14 @@ public protocol Tracer: LegacyTracer {
         file fileID: String,
         line: UInt
     ) -> Self.Span
+
+    /// Retrieve the recording span for the given `ServiceContext`.
```

Comment:
s/recording/active

Reply:
😬 Thanks for noticing. I'll create a follow-up to rename the comment.

Reply:
Fixed in #164